python/src/GridLayout_doc.i.in

```diff
@@ -108,3 +108,39 @@ Parameters
 ----------
 axes : bool
     *True* to draw the axes, *False* to hide the axes."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::GridLayout::getGraphCollection
+"Accessor to the collection of graphs.
+
+Returns
+-------
+coll : :class:`~openturns.Collection` of :class:`~openturns.Graph`
```

Comment:
sequence of :class:`~openturns.Graph`

Reply:
Nop. It is a Collection. You can call the getSize() method.

Reply:
But you might provide a list of `Graph` and let the `swig` do the job. Isn't it?

Reply:
not for the getter, only for the setter

Reply:
@sofianehaddad swig converts the arguments for us so we can use native python constructs to feed our methods, and we do our best in C++/swig to return objects that mimic their python counterparts. But they are still C++ objects with their specific methods and must be documented as it.